src/gazebo_motor_model.cpp

```diff
@@ -177,6 +180,10 @@ void GazeboMotorModel::VelocityCallback(CommandMotorSpeedPtr &rot_velocities) {
   } else ref_motor_rot_vel_ = std::min(static_cast<double>(rot_velocities->motor_speed(motor_number_)), static_cast<double>(max_rot_velocity_));
 }
 
+void GazeboMotorModel::MotorFailureCallback(const boost::shared_ptr<const msgs::Int> &fail_msg) {
```

Comment:
you can also document this function on the header file, above https://github.com/PX4/sitl_gazebo/pull/140/files#diff-8892233ba37c1996abc20f1a8ba5619cR152